neptune/experiments.py

```diff
@@ -500,6 +508,8 @@ def stop(self, exc_tb=None):
         if self._stderr_uploader:
             self._stderr_uploader.close()
 
+        sys.excepthook = sys.__excepthook__
```

Comment:
`pop_stopped_experiment` should be responsible for setting this variable when last experiment was popped, shouldn't it?